src/components/TheEditor.vue

```diff
@@ -57,90 +57,90 @@
     </v-tabs>
 
   </v-card>
-          
+
 </template>
 
 <script>
-  import _ from 'lodash'
-  import { md2adr, adr2md, ArchitecturalDecisionRecord } from '@/plugins/utilities'
+import lodash from 'lodash';
```

Comment:
Doesn't have lodash the convetoin to use `_`? (Hence the name low dash?) --> Should be commented, shoulddn't it? Even if a linter complains, just state that.

Reply:
I did't know that there was a conventions. Was changed again.